### PR TITLE
Adding 50+ new actions + 8 new triggers to the connector

### PR DESCRIPTION
**Pull request by RjImpx**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 1]
```diff
@@ -561,3226 +649,8098 @@
...
+                  "eventOrSessionCode"
+                ]
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:4330]: @RjImpx Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by RjImpx** [comment]
@sriyen-msft we have made the changes you requested. We noticed a build error because we changed an action however this action is new and is not available in the certified connector so not a breaking change

**Comment by RjImpx** [comment]
@sriyen-msft we have made a few fixes and also added more actions. Please review

**Comment by RjImpx** [comment]
@sriyen-msft The build errors indicates a breaking change however these are new actions and it is not published so our users have not created any workflows with this.

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 2]
```diff
@@ -423,3364 +491,12591 @@
...
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AwardNominationData"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: @RjImpx There are many empty response scehma in the swagger.  Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@RjImpx Apologies for the delay in response, We have a high number of submissions and had to prioritize the connectors which were ready for production deployment. We are reviewing submissions one by one sorted by created date time.

File: certified-connectors/Impexium/apiDefinition.swagger.json (hunk 2 above)
Comment (RjImpx) [reply certified-connectors/Impexium/apiDefinition.swagger.json]: @sriyen-msft Thank you. we updated the PR to make sure that there are no empty responses.We removed the schema property for those that do not have a response.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**